spec/features/collection_previews/table_of_contents_preview_spec.rb

```diff
@@ -6,10 +6,17 @@
         draft = create(:collection_draft, user: User.where(urs_uid: 'testuser').first)
         visit collection_draft_path(draft)
 
+        #add this to have screenshot image
+        #screenshot_and_open_image
         find('.tab-label', text: 'Additional Information').click
+
       end
 
       it 'displays a table of contents' do
+
```

Comment:
remove lines 16-18

Reply:
I think table_of_contents_preview is under Bryant's second half